setup.py

```diff
@@ -47,7 +47,6 @@ def get_extra_deps(rel_path):
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.7",
```

Comment:
do we need enforce python version? https://stackoverflow.com/questions/19534896/enforcing-python-version-in-setup-py

Reply:
https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires

Reply:
thanks! added